keyboards/viendi8l/keymaps/default/rules.mk

```diff
@@ -0,0 +1 @@
+VIA_ENABLE = yes
```

Comment:
VIA should not be enabled on the default keymap.

Reply:
Comitted in 6c7cc960f5